lib/optimizely/event_builder.rb

```diff
@@ -17,6 +17,7 @@
 require_relative './params'
 require_relative './version'
 require_relative '../optimizely/helpers/event_tag_utils'
+require 'securerandom'
```

Comment:
what is the need of it?

Reply:
To generate UUID for payload.
Ref:  https://github.com/optimizely/node-sdk/blob/27239d040553ba794b46fc086628203de166686e/lib/core/event_builder/index.js#L121